imap_processing/hi/hi_cdf_attrs.py

```diff
@@ -158,3 +159,54 @@
     var_type="support_data",
     variable_purpose="PRIMARY",
 )
+
+# Histogram attributes
+hi_hist_l1a_global_attrs = GlobalDataLevelAttrs(
+    data_type="L1A_CNT>Level-1A Histogram",
+    logical_source="imap_hi_l1a_hist",
+    logical_source_desc=("IMAP-HI Instrument Level-1A Histogram Data."),
+    instrument_base=hi_base,
+)
+
+hi_hist_l1a_angle_attrs = AttrBase(
+    validmin=GlobalConstants.INT_FILLVAL,
+    validmax=GlobalConstants.INT_MAXVAL,
```

Comment:
Are these supposed to be angles in degrees?
```suggestion
    validmin=0,
    validmax=360,
```